packages/js/customer-effort-score/src/components/feedback-modal/feedback-modal.tsx

```diff
@@ -19,7 +19,8 @@ import classnames from 'classnames';
  * @param {string}   props.isSubmitButtonDisabled Boolean to enable/disable the send button.
  * @param {string}   props.submitButtonLabel      Label for the send button.
  * @param {string}   props.cancelButtonLabel      Label for the cancel button.
- * @param {Function} props.onModalClose           Callback for when user closes modal by clicking cancel.
+ * @param {Function} props.onModalClose           Function to call when user closes modal by clicking X.
+ * @param {Function} props.onCancel               Function to call when user presses cancel.
```

Comment:
I do like you aren't the `Modal` word in the property. To me, it's redundant since the property belongs to the `<FeedbackModal />` component. I think we should rename also the `onModalClose` with simply `onClose` to be consistent
We can do that in a follow-up.

Reply:
Good idea! I'll tackle it in a follow-up PR